tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7771,8 +7771,12 @@ def err_hlsl_unsupported_for_version_lower : Error<
    "%0 is only allowed for HLSL %1 and lower.">;
 def err_hlsl_unsupported_keyword_for_min_precision : Error<
    "%0 is only supported with -enable-16bit-types option">;
-def err_hlsl_intrinsic_overload_in_wrong_shader_model : Error<
-   "overload of intrinsic %0 requires shader model %1 or greater">;
+def err_hlsl_derivatives_intrinsic_in_wrong_shader_kind : Warning<
+   "Derivatives intrinsic %0 only works in ps and cs_6.6+/as_6.6+/ms_6.6+/node">,
+    DefaultError, InGroup<HLSLIntrinsicCall>;
+def err_hlsl_intrinsic_overload_in_wrong_shader_model : Warning<
```

Comment:
These are now warnings, not errors, so they shouldn't start with the `err_` prefix.

Reply:
Fixed.